### PR TITLE
Feat: Create sign in/up modal

### PR DESCRIPTION
**Pull request by AhmadDuhoki**

### CODE REVIEW

File: src/tailwind.css [hunk 1]
```diff
@@ -3,6 +3,37 @@
...
+}
+
+.btn-darkBlue {
+  @apply bg-blue-800 text-white;
+}
+.btn-darkBlue:hover {
+  @apply bg-blue-700;
+}
+
+.btn-TransparentBlue {
```
Comment (wisammechano) [inline src/tailwind.css:24]: not sure why is this?

File: src/tailwind.css [hunk 2]
```diff
@@ -3,6 +3,37 @@
...
+  @apply bg-transparent text-black border-solid border-2 border-gray-500;
+}
+.btn-TransparentBlue:hover {
+  @apply bg-gray-200 border-solid border-2 border-gray-500;
+}
+
+.btn-redXBtn {
```
Comment (wisammechano) [inline src/tailwind.css:31]: this is also a one off component, and would make more sense if it was applied inside the RedXBtn.js directly

File: src/components/LineOrSVG.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          <line
+            x1="24.5"
+            y1="0.998264"
+            x2="25.5174"
+            y2="293.998"
+            stroke="black"
+            strokeOpacity="0.13"
+          />
+          <circle cx="25" cy="147" r="25" fill="#E9E9E9" />
+          <path
```
[the remark this answers is not in the document]
Comment (AhmadDuhoki) [reply src/components/LineOrSVG.js:23]: Hey Wissam!! I tried a lot but it was not showing the text inside the page.

[the remark this answers is not in the document]
Comment (wisammechano) [reply src/components/LineOrSVG.js]: No worries, I will leave it as is now and we can change later

**Pull request merged**